style.css

```diff
@@ -1,5 +1,15 @@
 body {
-    color: orange;
-    background-color: blue;
+    color: yellow;
+    background-color: rgb(76, 0, 255);
     font-family: 'Ranchers', cursive;
```

Comment:
Given this font, I would consider if you should increase the default `font-size` on the `p` element to help it be more readable. 👍🏻 The browser default is usually somewhere around 16-18 pixels, but this font is meant for more display purposes and needs a larger size to be readable by a wider audience.